qcodes/tests/dataset/test_subscribing.py

```diff
@@ -4,16 +4,49 @@
 
 import pytest
 from numpy import ndarray
+import logging
 
+import qcodes
 from qcodes.dataset.param_spec import ParamSpec
 # pylint: disable=unused-import
-from qcodes.tests.dataset.temporary_databases import (empty_temp_db,
-                                                      experiment,
-                                                      dataset)
+from qcodes.tests.dataset.temporary_databases import (
+    empty_temp_db, experiment, dataset)
+# pylint: enable=unused-import
+
+from qcodes.tests.test_config import default_config
+
+
+log = logging.getLogger(__name__)
 
 VALUE = Union[str, Number, List, ndarray, bool]
 
 
+class TestSubscriber():
```

Comment:
I would suggest not calling this class Test... since it's not actually a test. \

This works fine at the moment because pytest only discovers methods starting with `test` inside such a class and only if there is no __init__ method but it's still potentially confusing

Reply:
good point!